packages/sources/tradinghours/README.md

```diff
@@ -0,0 +1,44 @@
+# TRADINGHOURS
+
+![0.1.0](https://img.shields.io/github/package-json/v/smartcontractkit/external-adapters-js?filename=packages/sources/tradinghours/package.json) ![v3](https://img.shields.io/badge/framework%20version-v3-blueviolet)
```

Comment:
This version doesn't seem to match the version in package.json. Was this generated automatically?

Reply:
Yup the readme was outdated, just re-generated it!